pkg/ksync/cli.go

```diff
@@ -11,39 +11,58 @@ import (
 	"github.com/jenkins-x/jx/pkg/util"
 )
 
-// InstallKSync install ksync
-func InstallKSync() (string, error) {
+// KSyncer contains the methods used for ksync
+type KSyncer interface {
+	GetJXBinLocation() (string, error)
```

Comment:
I don't think this should go into this interface. We have already _JXBinaryLocation_ in the util package.